### ISSUE TITLE
Add titles to Second Brains /about API call

### ISSUE THREAD

**Issue by ecurrencyhodler** [stakwork/sphinx-js#12]
We are going to grab the title of the second brain through the `/about` call. Let's go through each of the 2B's to make sure they at least have a "title" listed in them so that when this call is made, a value is returned. 

Ideally the title should be 1-2 words. No longer than 3 words max.

**Comment by tomsmith8** [comment]
Reviewed the Secondbrains that we probably want on there to start:

- Bitcoin: https://knowledge-graph.sphinx.chat/about
- Machine Learning: https://boltwall.swarm20.sphinx.chat/about
- Keto & Carnivore: https://boltwall.swarm23.sphinx.chat/about

**Comment by ecurrencyhodler** [comment]
Perfect. Thanks tom. Are these all updated? Or do we still need to do that?

**Comment by tomsmith8** [comment]
Yes, they all have a variable `title` in the `/about` endpoint response e.g.
```
{"app_version":null,"description":"Learn about changes in Bitcoin","mission_statement":"Learn about changes in Bitcoin","search_term":"Bitcoin","title":"Bitcoin"}
```

**Comment by ecurrencyhodler** [comment]
Awesome ty.
